src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -363,7 +363,7 @@ export async function clearCellForExecution(editor: NotebookEditor, cell: Notebo
     await chainWithPendingUpdates(editor.document, (edit) => {
         const metadata = cell.metadata.with({
             statusMessage: undefined,
-            executionOrder: undefined,
+            executionOrder: null,
```

Comment:
A bug in VSC was fixed & we need to use `null` to reset existing execution order.